addons/udes_stock_cron/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+{
+    "name": "UDES Stock Cron",
+    "summary": "udes stock cron",
+    "description": "Core models and configuration for UDES Stock cron - Odoo 14",
+    "author": "Unipart digital",
+    "website": "http://github/unipartdigital/udes-open",
+    "category": "UDES",
+    "version": "0.1",
+    "depends": ["base", "stock", "stock_picking_batch", "udes_common", "udes_cron", "udes_stock"],
```

Comment:
Nit:
I think that list of depends module shouldnt be longer than needed:
I think that we add on depends modules if needed or if they are specifically somewhere mostly on xml or any external identifier called.
Example stock is not needed to add as we are adding udes_stock which depends on stock and by hierarchy makes udes_stock_cron too depending on stock, but in our example we need it as you are inheriting a view from stock and using stock.xml_id in inherited_id.
But maybe we dont need base as in fact all modules depend on base, maybe we dont need udes_common if udes_cron depends on it and we are not using the name of the module in udes_stock_cron.
Hope that i was clear